server/src/controllers/tasks/addTask.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response, NextFunction } from 'express';
+import { addTaskQuery } from '../../database';
+import { addTaskSchema } from '../../utils/validation';
+import { customError } from '../errors';
+
+const addTask = async (req: Request, res: Response, next: NextFunction) => {
+  const {
+    name, description, priority, endDate, status, sectionid,
+  } = req.body;
+  try {
+    await addTaskSchema.validateAsync({ name, description, endDate });
+    const { rows } = await addTaskQuery({
+      name,
+      description,
+      priority,
+      endDate,
+      status,
+      sectionid,
+    });
+    res.status(201).json({ data: rows[0], msg: ' Task added successfully!' });
```

Comment:
be consist please, here you should send msg or message ?

Reply:
Ok Nizar , Done